Users/Docker_Image.rst

```diff
@@ -17,6 +17,10 @@ that they're running on and only requires applications being shipped
 with things not already running on the host computer. This gives a
 significant performance boost and reduces the size of the application.
 
+.. note::
+
+   The Coala Docker image does not support Python 2 analysis.
```

Comment:
> coala is always written with a lower case c.

Reply:
Also please use the same indent everywhere, I think it's 2 spaces (please recheck that)

Reply:
Yeah its of two spaces.